packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -3,32 +3,50 @@
  * Licensed under the MIT License.
  */
 
-import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { ITelemetryLogger, ITelemetryProperties } from "@fluidframework/common-definitions";
 import { IConnectionDetails } from "@fluidframework/container-definitions";
 import { ConnectionMode, IQuorumClients, ISequencedClient } from "@fluidframework/protocol-definitions";
 import { PerformanceEvent } from "@fluidframework/telemetry-utils";
 import { assert, Timer } from "@fluidframework/common-utils";
 import { ConnectionState } from "./connectionState";
 
-export interface IConnectionStateHandler {
+export interface IConnectionStateHandlerProps {
```

Comment:
Why the name change as it just does not contain the properties.

Reply:
I found it confusing that `ConnectionStateHandler` doesn't implement `IConnectionStateHandler`.  I don't feel very strongly about it, so if you have a different name suggestion or would prefer I revert just lmk :)

Reply:
Renaming to IConnectionStateHandlerInputs and adding a doc comment